runelite-client/src/main/java/net/runelite/client/plugins/discord/DiscordGameEventType.java

```diff
@@ -262,7 +294,95 @@
+	REGION_FISHER_REALM("Fisher Realm", DiscordAreaType.REGIONS, 10569),
+	REGION_FISHING_GUILD("Fishing Guild", DiscordAreaType.REGIONS, 10293),
+	REGION_FISHING_PLATFORM("Fishing Platform", DiscordAreaType.REGIONS, 11509),
+	REGION_FORSAKEN_TOWER("The Forsaken Tower", DiscordAreaType.REGIONS, 5435),
+	REGION_FOSSIL_ISLAND("Fossil Island", DiscordAreaType.REGIONS, 14394, 14395, 14649, 14650, 14651, 14652, 14906, 14907, 14908, 15162, 15163, 15164),
+	REGION_FREMENNIK("Fremennik Province", DiscordAreaType.REGIONS, 10296, 10552, 10808, 10809, 11064, 10810, 10811),
+	REGION_FREMENNIK_ISLES("Fremennik Isles", DiscordAreaType.REGIONS, 9276, 9532),
+	REGION_FROZEN_TUNDRAS("Frozen Tundras", DiscordAreaType.REGIONS, 6204, 6205, 6461, 6717),
+	REGION_GORAKS_PLANE("Gorak's Plane", DiscordAreaType.REGIONS, 12115),
+	REGION_GRAND_EXCHANGE("Grand Exchange", DiscordAreaType.REGIONS, 12598),
+	REGION_HARMONY("Harmony Island", DiscordAreaType.REGIONS, 15148),
+	REGION_ICE_PATH("Ice Path", DiscordAreaType.REGIONS, 11322, 11323, 11578),
+	REGION_ICEBERG("Iceberg", DiscordAreaType.REGIONS, 10558, 10559, 10658),
+	REGION_ISAFDAR("Isafdar", DiscordAreaType.REGIONS, 8497, 8498, 8753, 8754, 9009, 9010),
+	REGION_ISLAND_OF_STONE("Island of Stone", DiscordAreaType.REGIONS, 9790),
+	REGION_JIGGIG("Jiggig" , DiscordAreaType.REGIONS, 9775),
+	REGION_KANDARIN("Kandarin", DiscordAreaType.REGIONS, 9014, 9263, 9519, 9524, 9527, 9776, 9777, 9778, 9783, 10037, 10294, 10549, 10550, 10551, 10804, 10805, 10807),
+	REGION_KARAMJA("Karamja" , DiscordAreaType.REGIONS, 10801, 10802, 11054, 11311, 11312, 11313, 11566, 11567, 11568, 11569, 11822),
+	REGION_KEBOS_LOWLANDS("Kebos Lowlands", DiscordAreaType.REGIONS, 4665, 4921, 5178),
+	REGION_KEBOS_SWAMP("Kebos Swamp", DiscordAreaType.REGIONS, 4664, 4920, 5174, 5175, 5176, 5430, 5431),
+	REGION_KHARAZI_JUNGLE("Kharazi Jungle", DiscordAreaType.REGIONS, 11053, 11309, 11565, 11821),
+	REGION_KHARIDIAN_DESERT("Kharidian Desert", DiscordAreaType.REGIONS, 12844, 12845, 12846, 12847, 12848, 13100, 13101, 13102, 13103, 13104, 13357, 13359, 13360, 13614, 13615, 13616),
+	REGION_KOUREND("Great Kourend", DiscordAreaType.REGIONS, 6201, 6457, 6713),
+	REGION_KOUREND_WOODLAND("Kourend Woodland", DiscordAreaType.REGIONS, 5942, 6197, 6453),
+	REGION_LAW_ALTAR("Law Altar", DiscordAreaType.REGIONS, 9803),
+	REGION_LIGHTHOUSE("Lighthouse", DiscordAreaType.REGIONS, 10040),
+	REGION_LITHKREN("Lithkren", DiscordAreaType.REGIONS, 14142, 14398),
+	REGION_MAX_ISLAND("Max Island", DiscordAreaType.REGIONS, 11063),
+	REGION_MIND_ALTAR("Mind Altar", DiscordAreaType.REGIONS, 11083),
+	REGION_MISTHALIN("Misthalin", DiscordAreaType.REGIONS, 12593, 12594, 12595, 12849),
+	REGION_MOLCH("Molch", DiscordAreaType.REGIONS, 5177),
+	REGION_MOLCH_ISLAND("Molch Island", DiscordAreaType.REGIONS, 5432),
+	REGION_MORYTANIA("Morytania", DiscordAreaType.REGIONS, 13619, 13620, 13621, 13622, 13876, 13877, 13879, 14133, 14134, 14135, 14389, 14390, 14391, 14645, 14647, 14641, 14897, 14898),
```

Comment:
14135 should be Fenkenstrain's Castle.
14641, 14897, and 14898 should be Icyene Graveyard.